### PR TITLE
Add constant and function to calculate the minimum challenge questions

### PR DESCRIPTION
**Pull request by JustinGaoF**
Add Const MinQuestionPerChallenge used to prepare the challenge questions

### CODE REVIEW

File: config/common.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"github.com/tron-us/go-btfs-common/protos/guard"
+	"strconv"
+
+	"github.com/tron-us/go-common/env"
+	"github.com/tron-us/go-common/log"
```
Comment (taiyangc) [inline config/common.go]: Use go-common/v2

File: go.mod [hunk 2]
```diff
@@ -6,6 +6,7 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/protobuf v1.3.2
 	github.com/libp2p/go-libp2p-core v0.0.6
+	github.com/tron-us/go-common v1.0.2
```
Comment (taiyangc) [inline go.mod]: Run `go mod tidy`. go-common v1 should not show up.

File: config/common.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"github.com/tron-us/go-btfs-common/protos/guard"
```
Comment (ShellyWEI) [inline config/common.go]: i will quote what eric's first comment left for me:
we use first import portion as build-in library like 'strconv';
and second portion is library of project itself, in this project it's 'gp-common-btfs'
and third portion would be third-party library like 'go-common', 'zap'

Not sure if we are going to stick with this format or not.

Comment (taiyangc) [reply config/common.go]: Yes, we should stick to it.

Comment (JustinGaoF) [reply config/common.go]: Done the change

**Pull request merged**